Gems/Camera/Code/Source/CameraSystemComponent.cpp

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
```

Comment:
update: The header has changed slightly to break up the lines a bit to help with clang-format (we have a 140 character line limit) you might need to check another file and copy over the latest header)